realio/chain.json

```diff
@@ -36,34 +36,46 @@
     },
     "codebase": {
       "git_repo": "https://github.com/realiotech/realio-network",
-      "recommended_version": "v0.8.0-rc4",
+      "recommended_version": "v0.8.1",
       "compatible_versions": [
-        "v0.8.0-rc4"
+        "v0.8.1"
       ],
       "binaries": {
-        "linux/amd64": "https://github.com/realiotech/realio-network/releases/download/v0.8.0-rc4/realio-network_0.8.0-rc4_Linux_x86_64.tar.gz",
-        "linux/arm64": "https://github.com/realiotech/realio-network/releases/download/v0.8.0-rc4/realio-network_0.8.0-rc4_Linux_arm64.tar.gz",
-        "darwin/amd64": "https://github.com/realiotech/realio-network/releases/download/v0.8.0-rc4/realio-network_0.8.0-rc4_Darwin_x86_64.tar.gz",
-        "darwin/arm64": "https://github.com/realiotech/realio-network/releases/download/v0.8.0-rc4/realio-network_0.8.0-rc4_Darwin_arm64.tar.gz",
-        "windows/amd64": "https://github.com/realiotech/realio-network/releases/download/v0.8.0-rc4/realio-network_0.8.0-rc4_Windows_x86_64.zip"
+        "linux/amd64": "https://github.com/realiotech/realio-network/releases/download/v0.8.1/realio-network_0.8.1_Linux_x86_64.tar.gz",
+        "linux/arm64": "https://github.com/realiotech/realio-network/releases/download/v0.8.1/realio-network_0.8.1_Linux_arm64.tar.gz",
+        "darwin/amd64": "https://github.com/realiotech/realio-network/releases/download/v0.8.1/realio-network_0.8.1_Darwin_x86_64.tar.gz",
+        "darwin/arm64": "https://github.com/realiotech/realio-network/releases/download/v0.8.1/realio-network_0.8.1_Darwin_arm64.tar.gz",
+        "windows/amd64": "https://github.com/realiotech/realio-network/releases/download/v0.8.1/realio-network_0.8.1_Windows_x86_64.zip"
       },
+      "cosmos_sdk_version": "0.46",
+      "consensus": {
+        "type": "tendermint",
+        "version": "0.34"
+      },
+      "ibc_go_version": "6.1.1",
       "genesis": {
         "genesis_url": "https://raw.githubusercontent.com/realiotech/mainnet/main/realionetwork_3301-1/genesis.json"
       },
       "versions": [
         {
-          "name": "v0.8.0-rc4",
```

Comment:
Can we please retain v0.8.0-rc4 as a compatible version? (Or else as it's own object if it's consensus-breaking)